tests/Oryx.RuntimeImage.Tests/Node/NodeRuntimeImageContainsRequiredPrograms.cs

```diff
@@ -60,7 +63,8 @@ public void NodeImage_Contains_ApplicationInsights(string nodeTag)
                 () =>
                 {
                     Assert.True(result.IsSuccess);
-                    Assert.Contains("applicationinsights@1.4.1", actualOutput);
+                    Assert.Contains("applicationinsights@", actualOutput);
```

Comment:
Curious why break this into two? Btw you can do `npm list -g --depth=0` to get the details too

Reply:
yeah .. just an extra copy paste statement .. I will remove